src/sage/rings/universal_cyclotomic_field.py

```diff
@@ -1281,7 +1282,7 @@ def minpoly(self, var='x'):
             construct the polynomial.
         """
         gap_p = libgap.MinimalPolynomial(libgap.eval("Rationals"), self._obj)
-        return QQ[var](QQ['x_1'](str(gap_p)))
+        return PolynomialRing(QQ, var)(PolynomialRing(QQ, 'x_1')(str(gap_p)))
```

Comment:
```suggestion
        from sage.rings.polynomial.polynomial_ring_constructor import PolynomialRing
        return PolynomialRing(QQ, var)(PolynomialRing(QQ, 'x_1')(str(gap_p)))
```
Plus this import time will be negligible here.